iocage/lib/Jail.py

```diff
@@ -384,6 +416,29 @@ def update_jail_state(self):
         except:
             self.jail_state = None
 
+    def _teardown_mounts(self):
+
+        mountpoints = list(map(
+            lambda mountpoint: f"{self.path}/root{mountpoint}",
+            [
+                "/dev/fd",
+                "/dev",
+                "/proc",
+                "/root/compat/linux/proc"
```

Comment:
There are usually quite a few more mountpoints then this depending on the jail. Basejails come to mind, but the user may have their own, so we should have a recursive `umount` as well.